tests/cross-language/node/features/support/decrypt.js

```diff
@@ -0,0 +1,59 @@
+const { Given, When, Then } = require('@cucumber/cucumber')
+const assert = require('assert');
+const asherah = require("asherah");
+const fs = require("fs");
+
+const fileDirectory = "/tmp/";
+const fileName = "node_encrypted";
+
+var encryptedPayload;
+var decryptedPayload;
+
+var adoDatabaseName = process.env.TEST_DB_NAME;
+var adoUsername = process.env.TEST_DB_USER;
+var adoPassword = process.env.TEST_DB_PASSWORD;
+var adoPort = process.env.TEST_DB_PORT;
+var adoConnectionString = adoUsername+":"+adoPassword+"@tcp(localhost:"+adoPort+")/"+adoDatabaseName+"?tls=false"
+
+Given('I have encrypted_data from {string}', async function (string) {
+    var payload = fs.readFileSync(fileDirectory + fileName, 'utf8');
```

Comment:
You need to use the `string` as the file name instead the `fileName`. This scenario will run for each of the language outputs.

Reply:
Good catch!